packages/test/README.md

```
@@ -0,0 +1,243 @@
+pnpm test:ui
+
+# 运行可见浏览器模式
+pnpm test:headed
+
+# 运行调试模式
+pnpm test:debug
+
+# 查看测试报告
+pnpm test:report
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)